Build FAILED error C2259: 'msr::airlib::RealMultirotorConnector': cannot instantiate abstract class

Build FAILED.

"C:\Projects\Roadrunner-Project\AirSim\AirSim.sln" (default target) (1) ->
"C:\Projects\Roadrunner-Project\AirSim\DroneServer\DroneServer.vcxproj" (default target) (6) ->
(ClCompile target) ->
  c:\projects\roadrunner-project\airsim\droneserver\main.cpp(80): error C2259: 'msr::airlib::RealMultirotorConnector':
cannot instantiate abstract class [C:\Projects\Roadrunner-Project\AirSim\DroneServer\DroneServer.vcxproj]

--- a/AirLib/include/vehicles/multirotor/controllers/RealMultirotorConnector.hpp
+++ b/AirLib/include/vehicles/multirotor/controllers/RealMultirotorConnector.hpp
@@ -70,6 +70,12 @@ public:
         throw std::logic_error("getSegmentationObjectID() call is only supported for simulation");
     }
 
+    virtual void printLogMessage(const std::string& message, std::string message_param = "", unsigned char severity = 0)  override
+    {
+        unused(message);
+        unused(message_param);
+        unused(severity);
+    }
 
 private:
     VehicleControllerBase* controller_;

